apps/web/pages/settings/security/impersonation.tsx

```diff
@@ -63,11 +64,11 @@ const ProfileImpersonationView = () => {
         }}>
         <div className="flex space-x-3">
           <Switch
-            defaultChecked={!user?.disableImpersonation}
             onCheckedChange={(e) => {
-              setValue("disableImpersonation", !e, { shouldDirty: true });
+              setValue("disableImpersonation", e, { shouldDirty: true });
```

Comment:
Are we meant to be removing !

This switch disables impersonation if set to true 

even tho the value coming from the switch is false 

false->disables impersonation
true(default)-> impersonation enabled

Reply:
@sean-brydon pushed some changes